app/controllers/tags_controller.rb

```diff
@@ -18,7 +18,11 @@ class TagsController < ApplicationController
   def index
     skip_authorization
     @tags_index = true
-    @tags = Tag.direct.includes(:sponsorship).order(hotness_score: :desc).limit(100)
+    @tags = if params.key?(:q) && params[:q].present?
```

Comment:
Suggestion: This seems almost too defensive, the `.present?` check alone seems to cover all relevant cases, doesn't it?

![Screen Shot 2022-01-25 at 11 13 48](https://user-images.githubusercontent.com/47985/150909722-834a7b9d-b766-481b-a3bd-92bd501ef31e.png)

```suggestion
    @tags = if params[:q].present?
```